packages/hooks/src/useApolloClient.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { invariant } from 'ts-invariant';
+import { getApolloContext } from '@apollo/react-common';
+
+export function useApolloClient() {
+  const { client } = React.useContext(getApolloContext());
+  invariant(
+    !client,
```

Comment:
👋 Unsolicited comment - so apologies in advance.

I believe the logical not operator here is causing the `useApolloClient` hook in release `@apollo/react-hooks@0.1.0-beta.6` to throw an error and not work. The `invariant` function itself places the `!` on the condition applied and seems to be used [appropriately elsewhere](https://github.com/apollographql/react-apollo/blob/15dea607a6caf71c0902c788fba43b323aabe12f/packages/common/src/context/ApolloProvider.tsx#L25).

Thanks for all the work you've done thus far! 🙏

Reply:
also please do `return client!;` or something as it's currently typed as `ApolloClient | undefined` 🙂

Reply:
@djirdehh @SimenB these should be fixed in `@apollo/react-hooks@0.1.0-beta.7`. Thanks!